.github/pull_request_template.md

```diff
@@ -0,0 +1,18 @@
+## Description
+
+Describe the content of the PR and links to related issues, bugs of features.
+
+## Requirements
+
+Before submitting this PR, please make sure:
+
+- [ ] You have followed the coding standards guidelines established at [Code Review Checklist](https://paper.dropbox.com/doc/Code-Review-Checklist--BD7zigBMAhMZAPkeNENeuU2UAg-IlsYffZgTwyKEylty7NhY).
```

Comment:
Great! In practice most people are unlikely to read the full checklist each time they write/review a PR, at least after the first few times. Because of this I've suggested below adding two points from that checklist explicitly to this one (adding the whole checklist here would be overkill).